src/wipplpy/modules/config_reader.py

```diff
@@ -10,10 +10,23 @@
 from WiPPLPy.modules.generic_get_data import lazy_get
 
 
-class ConfigReader:
+class MDSplusConfigReader:
```

Comment:
I'm thinking that this config reader also might read things for MySQL or whatever else but we can change it in the future if/when that is required!

Reply:
I see, then maybe I should make it more general. I had rewritten it as `MDSplusConfigReader` because everything inside of the class (and the default filepath itself) refers to MDSplus labels. Maybe call the INI file 'connection_config.INI' or 'database_config.INI'?

Reply:
I'd say let's leave it as MDSplusConfigReader for now. It's simple enough to change in the future if/when we need it.